### PR TITLE
PP-6982

### PR DESCRIPTION
**Pull request by stevenjmesser**
## WHAT
Updated the link as requested and added warning text.

## HOW 
Small frontend change, please do check it works :)

### CODE REVIEW

File: app/views/your-psp/_worldpay-flex.njk [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
 
 <p class="govuk-body">If you have set up 3DS Flex on your Worldpay account, you need to enter the following details to turn 3DS Flex on. You’ll find these details in your <a class="govuk-link" href="https://secure.worldpay.com/sso/public/auth/login.html">Worldpay account</a>.</p>
 
+<div class="govuk-warning-text">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+    If your credentials are wrong, you are not able to take payments.
+  </strong>
+</div>
```
Comment (alexbishop1) [inline app/views/your-psp/_worldpay-flex.njk:13]: We should use the Nunjucks macro for this:

```suggestion
{{ govukWarningText({
  text: "If your credentials are wrong, you are not able to take payments.",
  iconFallbackText: "Warning"
}) }}
```

Comment (stevenjmesser) [reply app/views/your-psp/_worldpay-flex.njk]: Ah, fantastic! I did not know about that.

**Review by alexbishop1 — APPROVED** [review]
Looks good!

image

**Comment by alexbishop1** [comment]
To have this change built, it will need to come from within alphagov, so closing in favour of https://github.com/alphagov/pay-selfservice/pull/2380.

**Pull request closed without merging**